app/views/admin/facilities/_form.html.erb

```diff
@@ -1,20 +1,30 @@
 <div class="row">
   <div class="col-md-6">
     <%= bootstrap_form_with(model: [:admin, facility_group, facility], local: true, autocomplete: "off", label_errors: true) do |form| %>
-      <%= form.text_field :name, id: :facility_name, required: true, autocomplete: "off" %>
-      <%= form.text_field :facility_type, id: :facility_type, required: true %>
-      <%= form.select :facility_size, Facility.facility_sizes.map { |key, value| [key.capitalize, value] }, id: :facility_type, required: true %>
-      <%= form.text_field :street_address, id: :facility_street_address %>
-      <%= form.text_field :village_or_colony, id: :facility_village_or_colony %>
+      <%= form.text_field :name, id: :facility_name, required: true, autocomplete: "off", help: "Include the facility type in the name. Example: PHC Brooklyn", label: "Facility name *" %>
+      <%= form.text_field :facility_type, id: :facility_type, help: "Examples: DH, SDH, CHC, PHC, UPHC, HWC, SC", label: "Facility type *", required: true %>
+      <%= form.select :facility_size, Facility.facility_sizes.map { |key, value| [key.capitalize, value] }, id: :facility_type, label: "Size *", help: "Select the closest relative size", required: true %>
+      <h3 class="mt-5">Address</h3>
+      <%= form.text_field :street_address, id: :facility_street_address, label: "House number and street" %>
```

Comment:
Does putting a `label` here continue to work well with locales? Currently, for example, in Bangladesh dashboards, the facility address fields in this form are picked up from [here](https://github.com/simpledotorg/simple-server/blob/master/config/locales/address/en_BD.yml) depending on `DEFAULT_PREFERRED_DASHBOARD_LOCALE` – worth checking.